### PR TITLE
[1.1] Add packaging for Fedora27, OpenSuse423, Ubuntu1804

### PR DESCRIPTION
**Pull request by ianhays**
cc: @ericstj @weshaggard @janvorli @dagood

### CODE REVIEW

**Comment by ianhays** [comment]
@janvorli @rakeshsinghranchi @weshaggard any comments on this one?

File: src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.builds [hunk 1]
```diff
@@ -63,6 +71,10 @@
       <OSGroup>Linux</OSGroup>
       <Platform>amd64</Platform>
     </Project>
+    <Project Condition="'$(TargetsLinux)' == 'true' and '$(DistroRid)' == 'ubuntu.18.04-x64'" Include="ubuntu/16.04/Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
```
Comment (janvorli) [inline src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.builds]: The include looks incorrect - it refers to 16.04 instead 18.04.

File: src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.builds [hunk 2]
```diff
@@ -47,6 +51,10 @@
       <OSGroup>Linux</OSGroup>
       <Platform>amd64</Platform>
     </Project>
+    <Project Condition="'$(TargetsLinux)' == 'true' and '$(DistroRid)' == 'opensuse.42.3-x64'" Include="opensuse/42.1/Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
```
Comment (janvorli) [inline src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.builds]: The include looks incorrect - it refers to 42.1 instead 42.3.

File: src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.builds [hunk 3]
```diff
@@ -39,6 +39,10 @@
       <OSGroup>Linux</OSGroup>
       <Platform>amd64</Platform>
     </Project>
+    <Project Condition="'$(TargetsLinux)' == 'true' and '$(DistroRid)' == 'fedora.27-x64'" Include="fedora/24/Microsoft.NETCore.Runtime.CoreCLR.pkgproj">
```
Comment (janvorli) [inline src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.builds]: The include looks incorrect - it refers to 24 instead 27.

File: src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/opensuse/42.3/Microsoft.NETCore.Runtime.CoreCLR.pkgproj [hunk 4]
```diff
@@ -0,0 +1,56 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <SkipPackageFileCheck>true</SkipPackageFileCheck>
+    <PackageTargetRuntime>opensuse.42.2-$(PackagePlatform)</PackageTargetRuntime>
```
Comment (janvorli) [inline src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/opensuse/42.3/Microsoft.NETCore.Runtime.CoreCLR.pkgproj]: This refers to 42.2, but the file is for 42.3.

File: src/.nuget/Microsoft.NETCore.TestHost/Microsoft.NETCore.TestHost.builds [hunk 5]
```diff
@@ -39,6 +39,10 @@
       <OSGroup>Linux</OSGroup>
       <Platform>amd64</Platform>
     </Project>
+    <Project Condition="'$(TargetsLinux)' == 'true' and '$(DistroRid)' == 'fedora.27-x64'" Include="fedora/24/Microsoft.NETCore.TestHost.pkgproj">
```
Comment (janvorli) [inline src/.nuget/Microsoft.NETCore.TestHost/Microsoft.NETCore.TestHost.builds]: In this file, the includes also seem to have distro version mismatch at multiple places.

**Pull request merged**